src/sentry/api/serializers/snuba.py

```diff
@@ -146,7 +146,10 @@ def zerofill(data, start, end, rollup, allow_partial_buckets=False):
         except IndexError:
             pass
 
-        rv.append((key, []))
+        if fill_default is None:
+            fill_default = []
+
+        rv.append((key, fill_default))
```

Comment:
Not 100% certain that I'm using this `zerofill` correctly actually.

Reply:
What is the purpose of `zerofill` and the fill_default parameter actually? I looked at this for a while and couldn't figure out.